client/src/components/AddTranslationForm.js

```diff
@@ -0,0 +1,119 @@
+import React, { Component } from "react";
+import { Link } from "react-router-dom";
+import {
+  Header,
+  Segment,
+  Container,
+  Button,
+  Form,
+  Message
+} from "semantic-ui-react";
+import { getDocumentById } from "../api/documents";
+import { putTranslation } from "../api/translations";
+import documentInformationBar from "./DocumentInformationBar";
+
+export class AddTranslationForm extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      document: [],
+      content: "",
+      submitErr: false,
+      isSend: false,
+      errorMessage: ""
+    };
+  }
+
+  componentDidMount() {
+    let documentId = this.props.match.params.documentId;
+    getDocumentById(documentId)
+      .then(document => {
+        this.setState({ document: document });
+      })
+      .catch(err => console.log(err));
```

Comment:
Instead of console.log here you could reuse your errorMessage state to display an error on the screen maybe?

Reply:
Done